src/app/dataset-flow/dataset-flow-details/tabs/flow-details-history-tab/flow-details-history-tab.helpers.ts

```diff
@@ -141,8 +141,20 @@ export class DatasetFlowDetailsHelpers {
                                                       : "blocks"
                                               }`
                                             : "Dataset is up-to-date";
+
+                                    case "FlowDescriptionDatasetHardCompacting":
+                                        switch (flowDetails.description.compactingResult?.__typename) {
+                                            case "FlowDescriptionHardCompactingSuccess":
+                                                return `There were ${flowDetails.description.compactingResult.originalBlocksCount} block(s), now there are ${flowDetails.description.compactingResult.resultingBlocksCount} block(s)`;
```

Comment:
Compacted N original blocks to M resulting blocks

Reply:
Changed